l10n_es_aeat_sii_oca/models/account_move.py

```diff
@@ -810,6 +811,23 @@ def _compute_sii_enabled(self):
             else:
                 invoice.sii_enabled = False
 
+    @api.model
+    def _search_sii_enabled(self, operator, value):
+        domain = super()._search_sii_enabled(operator, value)
+        invoice_types = self.get_sale_types() + self.get_purchase_types()
+        condition_1 = [("journal_id.sii_enabled", operator, value)]
+        condition_2 = [("fiscal_position_id.aeat_active", operator, value)]
+        search_ko = (operator == "=" and not value) or (operator == "!=" and value)
+        exp_condition = OR if search_ko else AND
+        if not search_ko:
+            condition_2 = OR([condition_2, [("fiscal_position_id", "=", False)]])
+        return AND(
+            [
+                [("move_type", "in", invoice_types)],
```

Comment:
Según entiendo, esto tiene que cambiar también si `sii_enabled != True` o `sii_enabled = False`, primero uniéndose con `OR` y no `AND`, y luego poniendo el operador `not in`.